tests/ert_tests/ert3/engine/integration/test_engine.py

```diff
@@ -189,6 +217,35 @@ def test_export_uniform_polynomial_evaluation(
     )
 
 
+@pytest.mark.requires_ert_storage
+def test_export_x_uncertainties_polynomial_evaluation(
+    workspace,
+    x_uncertainty_ensemble,
+    x_uncertainty_stages_config,
+    evaluation_experiment_config,
+    x_uncertainty_parameters_config,
+):
+    uni_dir = workspace / ert3.workspace.EXPERIMENTS_BASE / "uncertainty"
```

Comment:
As commented in the examples as well, should we call this `x_uncertainty` instead?